src/lib/components/ui/Alert/Alert.jsx

```diff
@@ -1,84 +1,56 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import getRootVariantClassName from '../../../helpers/getRootVariantClassName';
 import { withProviderContext } from '../../../provider';
 import styles from './Alert.scss';
 
-export const Alert = (props) => {
-  const {
-    children,
-    closeHandler,
-    icon,
-    id,
-    translations,
-    type,
-  } = props;
-
-  const rootTypeClass = (variant) => {
-    if (variant === 'success') {
-      return styles.isRootSuccess;
-    }
-
-    if (variant === 'error') {
-      return styles.isRootError;
-    }
-
-    if (variant === 'warning') {
-      return styles.isRootWarning;
-    }
-
-    if (variant === 'info') {
-      return styles.isRootInfo;
-    }
-
-    if (variant === 'help') {
-      return styles.isRootHelp;
-    }
-
-    return styles.isRootNote;
-  };
-
-  return (
+export const Alert = ({
+  children,
+  closeHandler,
+  icon,
+  id,
+  translations,
+  variant,
+}) => (
+  <div
+    className={[
+      styles.root,
+      getRootVariantClassName(variant, styles),
+    ].join(' ')}
+    id={id}
+    role="alert"
+  >
+    {icon && (
+      <div className={styles.icon}>
+        {icon}
+      </div>
+    )}
     <div
-      className={[
-        styles.root,
-        rootTypeClass(type),
-      ].join(' ')}
-      id={id}
-      role="alert"
+      className={styles.message}
+      {...(id && { id: `${id}__content` })}
     >
-      {icon && (
-        <div className={styles.icon}>
-          {icon}
-        </div>
-      )}
-      <div
-        className={styles.message}
-        {...(id && { id: `${id}__content` })}
-      >
-        {children}
-      </div>
-      {closeHandler && (
-        <button
-          type="button"
-          {...(id && { id: `${id}__close` })}
-          className={styles.close}
-          onClick={() => closeHandler()}
-          onKeyPress={() => closeHandler()}
-          tabIndex="0"
-          title={translations.close}
-        >
-          <span className={styles.closeSign}>×</span>
-        </button>
-      )}
+      {children}
     </div>
-  );
-};
+    {closeHandler && (
+      <button
+        type="button"
+        {...(id && { id: `${id}__close` })}
+        className={styles.close}
+        onClick={() => closeHandler()}
+        onKeyPress={() => closeHandler()}
+        tabIndex="0"
+        title={translations.close}
+      >
+        <span className={styles.closeSign}>×</span>
+      </button>
+    )}
+  </div>
+);
 
 Alert.defaultProps = {
   closeHandler: null,
   icon: null,
   id: undefined,
-  type: 'note',
```

Comment:
I am not sure if it is possible to have defaults in those cases. If it is empty, it should have neutral look and I also do not think we have this kind of variant and whether we want default variant of those components because they might not have real use-case. And creating default variant just to have "default appearance" does not sound good to me.

Please @adamkudrna let us know your opinion. I believe Adam has wider view into UI libraries so I will probably follow his recommendation in this case.

Reply:
I'm for 2 as @mbohal suggests. A default, unstyled appearance (like with Bootstrap) doesn't make sense to me because I don't see any use case for that, as @bedrich-schindler also says. Default appearance may be (and it often is) hidden in implementation, but from the design point of view, such a visual variant makes no sense and should not be suggested or made available to use. Let's go with sensible defaults per component.

Reply:
I'm proposing these `variant` defaults:

- `Alert`: `note` as the lowest (and neutral) visual priority available (`light` and `dark` are meant rather for special purposes).
- `Badge`: `note`, see above.
- `Button`: `primary` because there should be primary buttons (we think).
- `Card`: `null` which makes `variant` optional (correct from the design point of view).